app/src/main/java/com/android/sample/ui/profile/Profile.kt

```diff
@@ -117,7 +139,7 @@ fun ProfileScreen(user: ProfileData, navigationAction: NavigationActions) {
                     Button(
                         onClick = { /* Handle button click */},
                         modifier = Modifier.fillMaxWidth().testTag("editProfileButton"),
-                        colors = ButtonDefaults.buttonColors(containerColor = Color.White),
+                        colors = ButtonDefaults.buttonColors(containerColor = PrimaryWhite),
```

Comment:
Consider using the colors from the MaterialTheme.colorScheme to maintain consistency.

```suggestion
                        colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.background),
```